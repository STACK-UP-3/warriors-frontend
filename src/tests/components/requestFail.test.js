import React from 'react';
import { shallow } from 'enzyme';
import RequestFail from '../../components/requestFail';

describe('*************** Testing the Request Fail  component ***************',()=>{
    it('Should render the reset password page correctly', ()=>{
        const wrapper = shallow( <RequestFail  /> );
        expect(wrapper).toMatchSnapshot();
    });
});