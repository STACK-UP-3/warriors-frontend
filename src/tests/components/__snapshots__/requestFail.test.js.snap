// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`*************** Testing the Request Fail  component *************** Should render the reset password page correctly 1`] = `
<div>
  <h5>
    REQUEST FAILED
  </h5>
  <p>
    We are sorry your request could not be processed now. Please wait for the problem to be fixed we are currently working on it.
  </p>
</div>
`;
